packages/rfw/lib/src/flutter/runtime.dart

```diff
@@ -1095,6 +1437,17 @@ class _WidgetState extends State<_Widget> implements DataSource {
     return subscription.value;
   }
 
+  Object _widgetBuilderArgResolver(List<Object> rawDataKey) {
+    final _Key widgetBuilderArgKey = _Key(_kWidgetBuilderArgSection, rawDataKey);
+    final _Subscription subscription = _subscriptions[widgetBuilderArgKey] ??= _Subscription(
```

Comment:
putIfAbsent may be more idiomatic

Reply:
ACK. FWIW considered it, tried it but actually prefer this version. Let me know if you feel strongly about it.

Reply:
yeah, i often prefer this style too, to be honest.